docs/spa/index.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: doc
+title: SPA integration guide
+abstract: How to integrate Single Page Webapps with DID.app
+---
+
+Single Page WebApps (SPAs) can use DID.app can authenticate users via OpenID Connect.
+Authenticating End Users takes two steps.
+
+1. Redirect a user to DID.app with an authentication request from your application.
+2. Fetch the user information from the authentication response.
+
+Fetching user information for authenticated users can be achieved via two methods.
+
+1. Have an API endpoint on your application backend fetch an id token.
+2. Use PKCE and fetch the id token from within the web app.
+
+Use option 1. if you are using cookie based sessions.
+If using DID.app to send invite emails you will have to use option 1.
+
+### Create Authentication Request
+
+Redirect the End User to the authorization url: [https://auth.did.app/oidc/authorize](https://auth.did.app/oidc/authorize).
+Provide the details of the authentication request as query parameters.
+
```

Comment:
Presumably an SPA developer would know what to do here?  How do I send the end user to the URL with the params described?

Reply:
Yeah I think this is too framework specific to have an example here. This is where we need to start having Vue.js or other front end guides.

Reply:
Ah right ok. This wasn't clear is my feedback but now I know it's an appendix then that's quite helpful.  Worth making it more explicit?

Reply:
Fair enough :-)